bin/juxtaposer/README.md

```diff
@@ -0,0 +1,113 @@
+# Perf Test Agent
+
+## Description
+
+This tool is used to compare timing data between arbitrary number of similar
+services to evaluate differences in speed between a specified baseline backend
+and the other backends.
+
```

Comment:
This is still missing:

- Status of this tool (eg `alpha`)
- Description of what it does, exactly (eg "Defines a perf test agent that can be deployed alongside Secretless to run through the following test scenarios: ...")